src/pages/productListing/ProductListing.js

```diff
@@ -0,0 +1,17 @@
+import { Header, Filter, Footer, Products } from "./index";
+import "./productlisting.css";
+
+const ProductListing = () => {
+  return (
+    <main className="products-container">
+      <Header />
+      <div className="main-content-container">
+        <Filter />
+        <Products />
+        <Footer />
+      </div>
+    </main>
+  );
+};
+
+export default ProductListing;
```

Comment:
again a nitpick...but you may avoid using default exports.

Reply:
[here](https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a) is why

Reply:
why we should avoid default  exports ??